KebPOS/Services/ProductService.cs

```diff
@@ -176,5 +177,10 @@ public static List<Product> GetProducts()
             };
             return products;
         }
+
+        public List<Order> GetOrders()
+        {
```

Comment:
In this case we won't need to call GetOrders from the service. It adds an unnecessary layer for such small app. So do you mind calling the controller straight from the main menu and just show a list using console.writeline while we don't have a UI solution?